buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy

```diff
@@ -269,4 +270,17 @@ class IndexGenerator {
         return JsonOutput.toJson(result)
     }
 
+    @CompileDynamic
+    private static List<String> generateTags(GuideMetadata guide) {
+        [
+            guide.tags +
+            guide.languages.collect { it.toString() } +
```

Comment:
Please replace this with `guide.languages*.toString()`

Reply:
done